### DISCUSSION TITLE
How to deploy Apisix dashboard on sub path, such as https://xxx.com/apisix-dashboard/

### DISCUSSION THREAD

**Discussion by zerochina2020** [apache/apisix#6436] [Q&A]
apisix dashboard部署时，如何配置子路径并通过子路径访问，例如：https://xxx.com/dashboard/

**Comment by woyouyitouchenfeijv** [comment]
use dns 😂 。That's how we use the company domain name

**Reply by zerochina2020** [reply]
It is still common to use sub path when cannot apply for a domain

**Reply by woyouyitouchenfeijv** [reply]
how do you  visit apisix
